blockly-rtc/src/WorkspaceClient.js

```diff
@@ -51,51 +53,92 @@
  * client corresponds to.
  */
 export default class WorkspaceClient {
-    constructor(workspaceId, getEventsHandler, addEventsHandler) {
```

Comment:
Nit: Why is so much of this a diff?  It looks like maybe you have inconsistent sizing on your tabs, so that some are 2 spaces and some are 4.  I think this diff will be much smaller if you fix that.

Reply:
Yes, there is inconsistent indentation in this file. I fixed those inconsistencies in this PR, but it seems to create a lot of noise and hide the relevant changes. My plan is to go back to inconsistent formatting for this PR and upload another PR with format changes that I will merge here. The diff should be smaller now. Thanks for your patience.